packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -113,9 +113,9 @@ export function getBlockColors(
 		}
 
 		if ( ! isCustomColor ) {
-			const mappedColor = find( defaultColors, {
-				slug: value,
-			} );
+			const mappedColor = Object.values( defaultColors ?? {} ).find(
```

Comment:
This comes in through an externally exposed API, so we're properly handling objects and nullish values.